components/elm/src/main/elmfates_interfaceMod.F90

```diff
@@ -937,6 +951,7 @@ subroutine dynamics_driv(this, bounds_clump, top_as_inst,          &
             this%fates(nc)%bc_in(s)%hlm_harvest_catnames = harvest_varnames
             this%fates(nc)%bc_in(s)%hlm_harvest_units = wood_harvest_units
          end if
+         this%fates(nc)%bc_in(s)%site_area=col_pp%wtgcell(c)*grc_pp%area(g)*1e6_r8
```

Comment:
need units for 1e6 please. I'm guessing this is m2 per km2?

Reply:
Right.